filebeat/docs/inputs/input-container.asciidoc

```diff
@@ -3,6 +3,12 @@
 [id="{beatname_lc}-input-{type}"]
 === Container input
 
+deprecated:[7.16.0]
+
+The container input is deprecated. Please use the the <<filebeat-input-filestream,`filestream input`>> and its <<filebeat-input-filestream-parsers-container,`container`>> parser.
```

Comment:
I'm wondering if we should add an example configuration here, there are a couple of pitfalls when using Filestream to ingest container data:
- The ID must be unique
- Following symlinks needs to be enabled.

Reply:
+1